src/ShallowTestWrapper.js

```diff
@@ -37,7 +37,7 @@ export default class ShallowTestWrapper extends TestWrapper {
   }
 
   html () {
-    return this.wrapper.html()
+    return this.wrapper.debug().replace(/\n(\s*)/g, '')
```

Comment:
what's the `.replace` for?

Reply:
When I updated it to debug() a few of the html() tests were failing:

AssertionError: expected the node in <Fixture /> to be '<span id="child">Test</span>', but it was '<span id="child">\n  Test\n</span>'      

This change removed that extra padding and the \n's to get them to match what they use to be. I took the regex from the jest-enzyme link I posted in the #218.

Let me know if you'd rather it be the other way and I can update those tests and remove the regex!